utils/build-galaxy-release.sh

```diff
@@ -20,6 +20,10 @@ Options:
   -a          Add all files, no only files known to git repo
   -k          Keep build directory
   -i          Install the generated collection
+  -o <A.B.C>  Build offline without using git, using version A.B.C
+              Also enables -a
+  -p <path>   Use path for the installation of the generated collection
+              (use with -i)
```

Comment:
If `-p` requires `-i`, shouldn't it force its use?

I prefer that `-p` forces `-i` (in the same sense that `-o` forces `-a`) rather than having to remember to use both.

Reply:
Done